Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3-slim
```

Comment:
Huh, though apparently the overwhelming advice from #webdev is that it's super Python 3 time if you're starting a new project, which we are.